engineering-team/index.html

```diff
@@ -53,7 +53,7 @@ <h1 class="text-roboto text-capitalize pb-2">SEF Engineering Team</h1>
                             With our community being active through various mediums such as standup meetings, knowledge
                             sharing sessions and communication platform groups, you will soon be exposed to the startup
                             culture, the various practices and procedures involved with contributing to projects. </p>
-                        <a href="/join-us.html">
+                        <a href="https://handbook.sefglobal.org/engineering-team/team/">
```

Comment:
Can you add the `target="_blank" property to this anchor tag? @Sanjana0319 

```suggestion
                        <a href="https://handbook.sefglobal.org/engineering-team/team/" target="_blank">
```

Reply:
All done, check out.